### DISCUSSION TITLE
[Feature Request] 请问有公安备案的区域嘛

### DISCUSSION THREAD

**Discussion by larrystd** [YunYouJun/hexo-theme-starry#227] [General 大局为重]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by YunYouJun** [comment]
不明白，「公安备案的区域」所指含义。
本项目只是一个主题，不涉及备案相关内容。
